aptos-move/framework/src/natives/aggregator_natives/helpers_v2.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+use move_binary_format::errors::PartialVMResult;
+use move_vm_types::values::{Reference, Struct, StructRef, Value};
+
+const VALUE_FIELD_INDEX: usize = 0;
+/// Given a reference to `Aggregator` Move struct returns a field value at `index`.
```

Comment:
The comment seems off, plus `get_aggregator_field` is pub(crate) in helpers, we can reuse it?

Reply:
Done.